src/main/java/org/opentripplanner/middleware/auth/Auth0Users.java

```diff
@@ -242,33 +248,39 @@ private static String getAuth0Url() {
     }
 
     /**
-     * Get an Auth0 oauth token for use in mocking user requests by using the Auth0 'Call Your API Using Resource Owner
-     * Password Flow' approach. Auth0 setup can be reviewed here: https://auth0.com/docs/flows/call-your-api-using-resource-owner-password-flow.
-     * If the user is successfully validated by Auth0 a bearer access token is returned, which is extracted and returned
-     * to the caller. In all other cases, null is returned.
+     * Get an Auth0 oauth token response for use in mocking user requests by using the Auth0 'Call Your API Using Resource
+     * Owner Password Flow' approach. Auth0 setup can be reviewed here: https://auth0.com/docs/flows/call-your-api-using-resource-owner-password-flow.
+     * If token response is returned to calling methods for evaluation.
      */
-    public static String getAuth0Token(String username, String password) throws JsonProcessingException {
+    public static HttpResponse<String> getCompleteAuth0TokenResponse(String username, String password) {
```

Comment:
Both downstream methods calling this method simply parse it into a TokenHolder. Why not just handle that within this method and change its return type?

Reply:
Because `ApiUserController.authenticateAuth0User' will provide the status code (produced by Auth0) to the user in the event of failure. I could update this to a fixed status code (403?) and then make your suggested update?